MomsSpaghetti/Pages/Shared/_Layout.cshtml

```diff
@@ -13,18 +13,18 @@
     <header>
         <nav class="navbar navbar-expand-sm navbar-toggleable-sm navbar-light bg-white border-bottom box-shadow mb-3">
             <div class="container">
-                <a class="navbar-brand" asp-area="" asp-page="/Index">MomsSpaghetti</a>
+                <a class="navbar-brand" asp-area="" asp-page="/Index"><h3>MomsSpaghetti</h3></a>
```

Comment:
Increased font size

Reply:
Best to use css styling for fonts.

- h3 should be used to describe content, more so than font.
- h3 tag shouldn't be nested inside of an anchor tag.